_data/round-4/participants.yml

```diff
@@ -608,9 +608,9 @@ Currently working in Kerala Startup Mission as Technology Innovation Fellow. Int
   img: https://drive.google.com/uc?id=0B183CwLYKDrYcmoyXzVCbENPWXc
 
 - name: Sina Hamedian
-  location: Santa Cruz
-  github: shamedian
-  twitter: sina
```

Comment:
@mattdark  this PR looks good to be merged, just wondering what happened here? Do you know or was it a typo?

Reply:
Hi @betatim 
I don't know what happened. I only modified my profile information not those lines

Reply:
@mattdark - It looks like you've deleted @shamedian's information on lines 611, 612 & 613. It might be because your branch isn't up to date with the master branch? Please can you pull all changes from master into your branch to check?

Here's a link that might be helpful: https://help.github.com/articles/syncing-a-fork/